sdk/core/Azure.Core.TestFramework/src/RecordedTestBase.cs

```diff
@@ -344,23 +344,11 @@ protected internal override object InstrumentClient(Type clientType, object clie
             return base.InstrumentClient(clientType, client, preInterceptors);
         }
 
-        protected internal T InstrumentOperation<T>(T operation) where T: Operation
-        {
-            return (T) InstrumentOperation(typeof(T), operation);
-        }
-
         protected internal override object InstrumentOperation(Type operationType, object operation)
             => InstrumentOperationInternal(operationType, operation);
 
         protected object InstrumentOperationInternal(Type operationType, object operation, params IInterceptor[] interceptors)
-        {
-            var interceptorArray = interceptors.Concat(new IInterceptor[] { new GetOriginalInterceptor(operation), new OperationInterceptor(Mode == RecordedTestMode.Playback) }).ToArray();
-            return ProxyGenerator.CreateClassProxyWithTarget(
-                operationType,
-                new[] { typeof(IInstrumented) },
-                operation,
-                interceptorArray);
-        }
+            => InstrumentOperationInternal(operationType, operation, Mode == RecordedTestMode.Playback, interceptors);
```

Comment:
The only other code according to [code search](https://cs.github.com/Azure/azure-sdk-for-net?q=InstrumentOperationInternal) that calls this I could also change to just call the `ClientTestBase.InstrumentOperationInternal`, but I felt like this helped codify a `RecordedTestBase`-specific variant with the right condition. I'm not strongly in favor of keeping this or changing the seemingly sole caller either way, though.

/cc @JoshLove-msft @m-nash

Reply:
I think it is fine to leave like this.